src/theme.js

```diff
@@ -1,3 +1,26 @@
+// @flow
+
+export const COLORS = {
+  accent: '#efb534',
+  error: '#f44336',
+  outline: '#bbb',
+  outline2: '#eee',
+  primary: '#3c91a1',
+  primary2: '#238393',
+  primary3: '#1e717e',
+  secondary: '#fff',
+  text: '#43484d',
+  textFaded: '#bdc6cf',
+  textInverse: '#fff',
+};
+
+export const TYPOGRAPHY = {
```

Comment:
I think it might make more sense to use `heading`/`secondary`/`paragraph`/`small` instead of DOM elements. That way we use the intent instead of the element type.